python/GafferOFXUITest/NodeUITest.py

```
@@ -0,0 +1,52 @@
+
+import unittest
+
+import Gaffer
+import GafferUI
+import GafferUITest
+import GafferOFX
+import GafferOFXUI
+
+class NodeUITest( GafferUITest.TestCase ) :
+
+	def testLifetimes( self ) :
```

Ruff (0.8.2)

Missing return type annotation for public function `testLifetimes`

Add return type annotation: `None`

(ANN201)